src/components/filmDetailsPage/Page.vue

```diff
@@ -45,6 +47,25 @@ export default {
       logoFirstPart: LOGO_FIRST_PART,
       logoSecondPart: LOGO_SECOND_PART
     };
+  },
+  computed: {
+    ...mapGetters(["findMovieById"]),
+    movie() {
+      return this.findMovieById(this.currentMovieId);
+    },
+    middleText() {
+      return (
+        I18Y[LOCALE].FILMS_BY +
+        " " +
+        this.$options.filters.getOneGenre(this.movie.genres).toLowerCase() +
```

Comment:
Consider using a variable to hold this value and then returning a complete string using template literal.

Reply:
Fixed